projects/ion/src/lib/input/input.component.html

```diff
@@ -0,0 +1,44 @@
+<div
+  data-testid="inputDiv"
+  class="input"
+  [class.disabled]="disabled"
+  [class.invalid]="invalid"
+  [class.valid]="valid"
+>
+  <ion-icon
+    data-testid="icon-left"
+    *ngIf="iconDirection === 'left'"
+    type="trash"
+    style="padding-right: 8px"
```

Comment:
```suggestion
    style="padding-right: 8px"
```
Add class to this, because this was repeat.